aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/options/AWSCognitoAuthWebUISignInOptions.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth.cognito.options;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.util.ObjectsCompat;
+
+import com.amplifyframework.auth.options.AuthWebUISignInOptions;
+import com.amplifyframework.util.Immutable;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Cognito extension of web ui sign in options to add the platform specific fields.
+ */
+public final class AWSCognitoAuthWebUISignInOptions extends AuthWebUISignInOptions {
+    private final String idpIdentifier;
+    private final String federationProviderName;
+
+    /**
+     * Advanced options for signing in via a hosted web ui.
+     * @param scopes specify OAUTH scopes
```

Comment:
nit: "OAuth"

Reply:
Alright I'll update all these in my next PR